docs/release-notes.md

```diff
@@ -496,1026 +497,6 @@ v3 releases.
-* Approx can be used to compare values against strong typedefs (available in C++11 mode only).
-  * Strong typedefs mean types that are explicitly convertible to double.
-* CHECK macro no longer stops executing section if an exception happens.
-* Certain characters (space, tab, etc) are now pretty printed.
-  * This means that a `char c = ' '; REQUIRE(c == '\t');` would be printed as `' ' == '\t'`, instead of ` == 9`.
-
-##### Fixes:
-* Text formatting no longer attempts to access out-of-bounds characters under certain conditions.
-* THROW family of assertions no longer trigger `-Wunused-value` on expressions containing explicit cast.
-* Breaking into debugger under OS X works again and no longer required `DEBUG` to be defined.
-* Compilation no longer breaks under certain compiler if a lambda is used inside assertion macro.
-
-##### Other:
-* Catch's CMakeLists now defines install command.
-* Catch's CMakeLists now generates projects with warnings enabled.
-
-
-### 1.6.x
-
-#### 1.6.1
-
-##### Features/ Changes:
-* Catch now supports breaking into debugger on Linux
-
-##### Fixes:
-* Generators no longer leak memory (generators are still unsupported in general)
-* JUnit reporter now reports UTC timestamps, instead of "tbd"
-* `CHECK_THAT` macro is now properly defined as `CATCH_CHECK_THAT` when using `CATCH_` prefixed macros
-
-##### Other:
-* Types with overloaded `&&` operator are no longer evaluated twice when used in an assertion macro.
-* The use of `__COUNTER__` is suppressed when Catch is parsed by CLion
-  * This change is not active when compiling a binary
-* Approval tests can now be run on Windows
-* CMake will now warn if a file is present in the `include` folder but not is not enumerated as part of the project
-* Catch now defines `NOMINMAX` and `WIN32_LEAN_AND_MEAN` before including `windows.h`
-  * This can be disabled if needed, see [documentation](configuration.md#top) for details.
-
-
-#### 1.6.0
-
-##### Cmake/ projects:
-* Moved CMakeLists.txt to root, made it friendlier for CLion and generating XCode and VS projects, and removed the manually maintained XCode and VS projects.
-
-##### Features/ Changes:
-* Approx now supports `>=` and `<=`
-* Can now use `\` to escape chars in test names on command line
-* Standardize C++11 feature toggles
-
-##### Fixes:
-* Blue shell colour
-* Missing argument to `CATCH_CHECK_THROWS`
-* Don't encode extended ASCII in XML
-* use `std::shuffle` on more compilers (fixes deprecation warning/error)
-* Use `__COUNTER__` more consistently (where available)
-
-##### Other:
-* Tweaks and changes to scripts - particularly for Approval test - to make them more portable
-
-
-## Even Older versions
-Release notes were not maintained prior to v1.6.0, but you should be able to work them out from the Git history
-
----
-
-[Home](Readme.md#top)
+visit: [Released Note](https://github.com/catchorg/Catch2/blob/devel/docs/release-note-in-detail.md)
```

Comment:
is this file committed?

Reply:
I think so.

Reply:
Okay, so there's the problem. When I try to convert it to vitepress, it failed since this is too long, so I use a visit thing. Than, I try to use `visit` to redirect user to the early one, but hard to maintain when more release note flows in. Also, it is not a relative url.
So, I want to use the approach I've use in `command-line.md`, redirecting user to a github md environment, which supports long markdown file.

Reply:
specify paths for maintainers, redirects for the users.

```md
# Command Line

visit [Command Line](https://github.com/catchorg/Catch2/blob/devel/docs-others/command-line.md)

stored in `./docs-others`.


<script>
export default {
    mounted() {
        window.location.replace("https://github.com/catchorg/Catch2/blob/devel/docs-others/command-line.md");
    }
}
</script>
```